Kudu.Core/Functions/FunctionManager.cs

```diff
@@ -417,8 +417,41 @@ private string GetFunctionLogPath(string name)
                 }
             }
 
-            return await FileSystemHelpers.ReadAllTextFromFileAsync(testDataFilePath);
+            // TODO use Rabin-Karp
```

Comment:
I would expect string.Contains() to already use a decently optimized implementation. Not worth getting fancier here.